src/pages/login/index.js

```diff
@@ -23,8 +23,8 @@ const Login = () => {
   const { push } = useHistory();
   const [login, { isLoading, isSuccess, error }] = useLoginMutation();
   const { authenticated, user } = useAuth();
-  const [data, setData] = useState({});
-  const [picture, setPicture] = useState('');
+  const [, setData] = useState({});
+  const [, setPicture] = useState('');
```

Comment:
Shouldn't this be `[picture, setPicture]`? Same for the data one

Reply:
Done